server/apps/server/src/player.erl

```diff
@@ -221,3 +218,14 @@ logout(State) ->
             Pid ! {error, Message},
             exit(2)
     end.
+
+update_user_state(Pid,
+                  Connection,
+                  #{email := Email,
+                    username := Username,
+                    name := Name}) ->
+    #user_state{pid = Pid,
+                connection = Connection,
+                email = Email,
+                username = Username,
+                name = Name}.
```

Comment:
Given that this function is being used only once, I'm not sure there are gains in not doing it inlined. Inlining it would not make it much better, so if you prefer to keep it, ok.

Reply:
fixed